jetstream/tools/maxtext/model_ckpt_conversion.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -ex
+
+idx=$(date +%Y-%m-%d-%H-%M)
+# Modify the `MODEL` and `MODEL_VARIATION` based on the model you use.
+export MODEL=$1
+export MODEL_VARIATION=$2
+export MODEL_NAME=${MODEL}-${MODEL_VARIATION}
+
+# After downloading checkpoints, copy them to GCS bucket at $CHKPT_BUCKET \
+# Please use seperate GCS paths for uploading model weights from kaggle ($CHKPT_BUCKET) and MaxText compatible weights ($MODEL_BUCKET).
```

Comment:
NIT: kaggle is gemma specific but not for llama?

Reply:
Updated.